### PR TITLE
Add data-ember-action pointer to default css

### PR DESCRIPTION
**Pull request by knownasilya**
Seems like a nice default. Thoughts?

### CODE REVIEW

**Comment by rwjblue** [comment]
It seems like a reasonable change, but I generally think we shouldn't ship with default styles.  It is unlikely that everyone will want it...

Maybe this could be a nice little addon, that does `app.import` for you...

**Comment by knownasilya** [comment]
@rwjblue I think anyone that know what they are doing, is usually adding a css preprocess so they delete the `app.css` anyway. This is more for those new beginners.

**Comment by lukemelia** [comment]
This particular css rule may be the exception, because without it, actions don't work on iOS with when they are on elements that are not naturally clickable. The only reason you wouldn't want something like this is if you are not supporting iOS, or only using action helpers on divs or buttons. If we keep it, it should be accompanied by short comment explaining why it exists.

**Comment by stefanpenner** [comment]
> If we keep it, it should be accompanied by short comment explaining why it exists.

yes, lets add a comment.

File: blueprints/app/files/app/styles/app.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/* 
+ * For enabling pointer cursor on all elements with an action.
+ *
+ * Required if expecting using action on non-buttons,
+ * and expecting to work in iOS.
+ */
```
Comment (knownasilya) [inline blueprints/app/files/app/styles/app.css]: @stefanpenner @lukemelia how's that?

Comment (stefanpenner) [inline blueprints/app/files/app/styles/app.css]: LGTM
@lukemelia r?

File: blueprints/app/files/app/styles/app.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/* 
+ * For enabling pointer cursor on all elements with an action.
+ *
+ * Required if expecting to use action on non-buttons,
+ * and expecting to work in iOS.
+ */
```
Comment (knownasilya) [inline blueprints/app/files/app/styles/app.css]: @stefanpenner @lukemelia how's that?

**Comment by stefanpenner** [comment]
so I think actually `for enabling pointer cursor` is no more documenting then the css itself. We should likely just say, something along the lines of: this makes touch events work for all elements with ember actions.

**Comment by knownasilya** [comment]
@stefanpenner updated

**Comment by stefanpenner** [comment]
@knownasilya nice, lets let @lukemelia who is likely better written and spoken then both of us combined give us input :) Then LGTM.

I'll get this out with 0.2.2

**Comment by rwjblue** [comment]
I agree with @lukemelia in concept, but I still believe that this should be an addon that uses `app.import`.  

The following would be upsides to using an addon instead of shipping this as a default `app.css`:

* It makes upgrades massively easier for folks
* Enables this CSS to be added even when using Sass, LESS, etc (because the addon would still be adding to `vendor.css` in those cases).
* Makes adding/removing as simple as modifying your `package.json`.

**Comment by rwjblue** [comment]
> Probably not worthwhile to create an addon, for one small rule

Addons do not have to be "heavy-weight" or a burden, they can be as simple as two files in a repo. We need to embrace our new addon overlords...

**Comment by stefanpenner** [comment]
add-ons can also be versioned, this code-snippet cannot.

**Comment by knownasilya** [comment]
@rwjblue those are all good points, especially the upgrade ones.

**Comment by knownasilya** [comment]
I think I'm going to close this in favor of an addon, although would we add that to the core addons?

**Comment by stefanpenner** [comment]
@knownasilya i think so, but @rwjblue should give us his input.

**Comment by stefanpenner** [comment]
closing in-favor of an add-on.

**Pull request closed without merging**